### PR TITLE
OHRI-2109  Add TB Prevention link under Tuberculosis program in the clinical per…

### PR DESCRIPTION
**Pull request by lucyjemutai**
…spectives

## Requirements

Add TB Prevention link under Tuberculosis program in the clinical perspectives

## Summary
Add TB Prevention link under Tuberculosis program in the clinical perspectives
-The link will expose a dashboard named Tuberculosis Prevention as displayed below:
-The TB Prevention dashboard contains a tile and a table

## Screenshots
image

### CODE REVIEW

**Comment by lucyjemutai** [comment]
@CynthiaKamau @ODORA0 , Please help in reviewing this pr. Thanks

**Review by ODORA0 — APPROVED** [review]

File: packages/esm-tb-app/src/dashboard.meta.tsx [hunk 1]
```diff
@@ -48,9 +48,15 @@ export const tbClinicalViewDashboardMeta = {
...
+  config: { columns: 1, type: 'grid', programme: 'tb', dashboardTitle: 'TB Treatment' },
   title: 'TB Treatment',
-  dashboardIcon: PillsAdd,
+};
+
+export const tptCasesDashboardMeta = {
+  name: 'tb-prevention-cases',
+  slot: 'tpt-cases-dashboard-slot',
+  config: { columns: 1, type: 'grid', programme: 'tpt', dashboardTitle: 'TB Prevention'},
+  title: 'TB Prevention',
```
Comment (pirupius) [inline packages/esm-tb-app/src/dashboard.meta.tsx:59]: Let's align the naming here. If this is `TB` then let the const naming and slots reflect that. Replace `tpt` with `tb`

The constant naming should also reflect `tbPreventionDashboardMeta`

**Review by pirupius — CHANGES_REQUESTED** [review]

File: packages/esm-tb-app/src/dashboard.meta.tsx (hunk 1 above)
Comment (ODORA0) [reply packages/esm-tb-app/src/dashboard.meta.tsx:59]: @lucyjemutai So I think you should maintain the tpt naming as we spoke about yesterday.
@pirupius The title according to the wireframe is TB Prevention, having it tpt will make it easier to tell apart from TB work itself

File: packages/esm-tb-app/src/dashboard.meta.tsx [hunk 2]
```diff
@@ -48,9 +48,15 @@ export const tbClinicalViewDashboardMeta = {
...
   slot: 'tb-cases-dashboard-slot',
-  config: { columns: 1, type: 'grid', programme: 'tb', dashboardTitle: 'TB Treatment', icon: PillsAdd },
+  config: { columns: 1, type: 'grid', programme: 'tb', dashboardTitle: 'TB Treatment' },
   title: 'TB Treatment',
-  dashboardIcon: PillsAdd,
+};
+
+export const tbPreventionDashboardMeta = {
+  name: 'tpt-prevention-cases',
+  slot: 'tpt-cases-dashboard-slot',
```
Comment (pirupius) [inline packages/esm-tb-app/src/dashboard.meta.tsx:57]: These were supposed to be aliged so name and slot shoule have `tb`

File: packages/esm-tb-app/src/routes.json [hunk 3]
```diff
@@ -197,6 +197,52 @@
...
+        "slot": "tpt-cases-dashboard-slot",
+        "config": {
+          "columns": 1,
+          "type": "grid",
+          "programme": "tpt",
+          "dashboardTitle": "TB Prevention"
+        },
+        "title": "TB Prevention"
+      }
+    },
```
Comment (pirupius) [inline packages/esm-tb-app/src/routes.json:216]: These haven't been aligned

**Review by pirupius — APPROVED** [review]

**Pull request merged**